test/atoms/AtomGraph_tests.jl

```diff
@@ -58,7 +58,7 @@ using Xtals
         ag2 = deserialize(abspath(@__DIR__, "..", "test_data", "strucs", "testgraph.jls"))
         @test adjacency_matrix(ag.graph) == adjacency_matrix(ag2.graph)
         @test elements(ag) == ag2.elements
-        @test Atoms.normalized_laplacian(ag) == ag2.laplacian
+        @test ChemistryFeaturization.Atoms.normalized_laplacian(ag) == ag2.laplacian
```

Comment:
Could you please expand on what exactly the namespace conflict is? I don't recall seeing any such conflicts while testing with the `OFD` PR.

It might also help to actually make a note of it for future reference.

Reply:
it's because Xtals also defines `Atoms` so while this worked before:
```
using ChemistryFeaturization.Atoms
# then later...
Atoms.whatever()
```
that's now ambiguous as to which `Atoms` it's referring to, so now I just have
```
using ChemistryFeaturization
# then later...
ChemistryFeaturization.Atoms.stuff...
```

Reply:
Ah, I see. In that case, I can't think of any other long term solution that's better than what we have rn.

Reply:
yeah I mean short of renaming our `Atoms` module to something else there isn't really another option